src/index.html

```diff
@@ -13,6 +13,38 @@
     />
```

Comment:
The stylesheet link is pointing to a SCSS file (styles/index.scss). Typically, browsers cannot directly interpret SCSS files. You should compile the SCSS to CSS and link to the resulting CSS file instead.